### ISSUE TITLE
Waiting on lock_cachedb with tiered cache configuration

### ISSUE THREAD

**Issue by DrDaveD** [cvmfs/cvmfs#2753]
I'm submitting this issue here on behalf of @duncanmmacleod because I've been trying to help him with this and he doesn't have access to the CERN jira.

He's been having a problem on his HPC system where users of ligo.osgstorage.org doing normal access of files are blocked with the following message in /var/log/messages:
```
(ligo.osgstorage.org) another process holds /var/lib/cvmfs/lock_cachedb, waiting.
```

Their system has a configuration made by a previous system administrator including the following tiered cache configuration in `/etc/cvmfs/domain.d/osgstorage.org.local`:
```
$ cat /etc/cvmfs/domain.d/osgstorage.org.local
# Set the local directory for storing special files (defaults to the cache directory).
CVMFS_WORKSPACE=/var/lib/cvmfs
CVMFS_QUOTA_LIMIT=10000
# Tiered Cache The tiered cache manager combines two other cache manager instances as
# an upper layer and a lower layer into a single functional cache manager.
# Usually, a small and fast upper layer (SSD, memory) is combined with a larger
# and slower lower layer (HDD, network drive).
CVMFS_CACHE_PRIMARY=hpc
#
CVMFS_CACHE_hpc_TYPE=tiered
CVMFS_CACHE_hpc_UPPER=disk
CVMFS_CACHE_hpc_LOWER=scratch
CVMFS_CACHE_hpc_LOWER_READONLY=no
#posix Uses a cache directory with the standard cache implementation
CVMFS_CACHE_disk_TYPE=posix
CVMFS_CACHE_disk_BASE=/var/lib/cvmfs/
# An "alien cache" provides the possibility to use a data cache outside the control of CernVM-FS.
# It is safe to have the alien directory shared by multiple CernVM-FS processes and it is safe to unlink files from the alien cache directory anytime.
# Since the alien cache is unmanaged, there is no automatic quota management provided by CernVM-FS;
# the alien cache directory is ever-growing. The CVMFS_ALIEN_CACHE requires CVMFS_QUOTA_LIMIT=-1 and CVMFS_SHARED_CACHE=no.
CVMFS_CACHE_scratch_TYPE=posix
CVMFS_CACHE_scratch_ALIEN=/scratch/LIGO/cvmfs
CVMFS_CACHE_scratch_SHARED=no
CVMFS_CACHE_scratch_QUOTA_LIMIT=-1
```

/scratch is a lustre filesystem they're sharing between nodes. The cvmfs client version is 2.7.3.  In addition to ligo.osgstorage.org, they also use gwosc.osgstorage.org.  

Is there something wrong with the configuration?  I don't know much about how the tiered cache system works.

**Comment by jblomer** [comment]
It is strange that it blocks on /var/lib/cvmfs/lock_cachedb. The configuration set the _base_ directory of the upper layer to `CVMFS_CACHE_disk_BASE=/var/lib/cvmfs/`, so the final cache directory should be `/var/lib/cvmfs/shared`.

Can you send the output of `cvmfs_config showconfig ligo.osgstorage.org` for the complete configuration? I can then try to reproduce the problem.

**Comment by DrDaveD** [comment]
@duncanmmacleod I see this issue from July 2021 has been languishing waiting for you to reply with the `showconfig` configuration of the machine with the tiered cache.  Can you still get it?

**Comment by vvolkl** [comment]
Closing due to inactivity. Feel free to open a new issue if you see this issue again.
